test/e2e/index.js

```diff
@@ -23,6 +23,8 @@ const chaiResponseValidator = require('chai-openapi-response-validator');
 
 const { expect } = chai;
 
+const logs = require('./logs.test');
```

Comment:
Not sure what the added value of this test is as it doesn't test really test anything.